internal/plugin/server.go

```diff
@@ -497,6 +505,18 @@ func (plugin *NvidiaDevicePlugin) apiDevices() []*pluginapi.Device {
 	return plugin.rm.Devices().GetPluginDevices()
 }
 
+// updateResponseForEnvVars sets the response when using the envvar device list strategy.
+func (plugin *NvidiaDevicePlugin) updateResponseForEnvVars(response *pluginapi.ContainerAllocateResponse, deviceIDs ...string) {
+	plugin.updateResponseForDeviceListEnvVar(response, deviceIDs...)
+	var channelIDs []string
+	for _, channel := range plugin.imexChannels {
+		channelIDs = append(channelIDs, channel.ID)
+	}
+	if len(channelIDs) > 0 {
+		response.Envs[spec.ImexChannelEnvVar] = strings.Join(channelIDs, ",")
+	}
+}
+
```

Comment:
Instead of hiding this in a single function, can we just create a separate one for the IMEX injection and call them both above, i.e.:
```
	if plugin.deviceListStrategies.Includes(spec.DeviceListStrategyEnvVar) {
		plugin.updateResponseForDeviceListEnvVar(response, deviceIDs...)
		plugin.updateResponseForImexChannelEnvVar(response, plugin.imexChannels...)
	}
```